grails-app/services/com/unifina/service/CommunityProductService.groovy

```diff
@@ -0,0 +1,89 @@
+package com.unifina.service
+
+import com.unifina.api.ApiException
+import com.unifina.api.CommunityJoinRequestCommand
+import com.unifina.api.NotFoundException
+import com.unifina.api.UpdateCommunityJoinRequestCommand
+import com.unifina.domain.community.CommunityJoinRequest
+import com.unifina.domain.community.CommunitySecret
+import com.unifina.domain.security.IntegrationKey
+import com.unifina.domain.security.SecUser
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder
+import org.springframework.security.crypto.password.PasswordEncoder
+
+class CommunityProductService {
+	PasswordEncoder encoder = new BCryptPasswordEncoder()
```

Comment:
We discussed this in the call, decision was: let's go with plaintext secrets in the first stage to make it similar to how API keys are handled. Later in a separate ticket let's upgrade the security of all secrets and store them in hashed form.